irods/user.py

```diff
@@ -48,6 +56,13 @@ def remove(self):
     def members(self):
         return self.manager.getmembers(self.name)
 
+    @property
+    def metadata(self):
+        if not self._meta:
+            self._meta = iRODSMetaCollection(
+                 self.manager.sess.metadata, User, self.name)
```

Comment:
did we agree this should be `iRODSUserGroup` instead of `User`?

Reply:
Not entirely clear, since every iRODS group is also a user; also, `iquest attrs` reveals there are META_USER_ATTR_* tags, but no META_GROUP_ATTR_* tags.